chungara-revista-de-antropologia-chilena.csl

```diff
@@ -1,205 +1,360 @@
 <?xml version="1.0" encoding="utf-8"?>
-<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" demote-non-dropping-particle="sort-only" sort-separator=", " et-al-use-first="2" version="1.0" default-locale="es-CL">
-  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" and="text" delimiter-precedes-et-al="never" delimiter-precedes-last="never" initialize-with="." demote-non-dropping-particle="sort-only" default-locale="es">
   <info>
-    <title>Chungara Revista de Antropología Chilena (Spanish - Chile)</title>
+    <title>Chungara Revista de Antropología Chilena</title>
     <title-short>Chungara</title-short>
     <id>http://www.zotero.org/styles/chungara-revista-de-antropologia-chilena</id>
     <link href="http://www.zotero.org/styles/chungara-revista-de-antropologia-chilena" rel="self"/>
-    <link href="http://www.zotero.org/styles/journal-of-ethnobiology" rel="template"/>
+    <link href="http://www.zotero.org/styles/society-for-american-archaeology" rel="template"/>
     <link href="http://chungara.cl/index.php/es/instrucciones-para-autores" rel="documentation"/>
     <author>
       <name>Erik Marsh</name>
+      <email>erik.marsh@gmail.com</email>
     </author>
     <category citation-format="author-date"/>
     <category field="anthropology"/>
-    <issn>0716-1182</issn>
-    <eissn>0717-7356</eissn>
```

Comment:
@e42mercury, did you remove the ISSNs by accident?

Reply:
I did! please put them back in. thanks for catching that.